content/en/providers/check-in/sp/_index.md

```diff
@@ -1021,6 +1025,48 @@ connected to Check-in.
 
 <!-- markdownlint-enable line-length no-inline-html -->
 
+#### 10.1 Capabilities
+
+<!-- markdownlint-disable line-length no-inline-html -->
+
+|          attribute name | Capabilities                                                                        |
+| ----------------------: | :---------------------------------------------------------------------------------- |
+|         **description** | This attribute describes the resource or child-resource a user is allowed to access |
```

Comment:
.. to access, optionally specifying certain actions the user is entitled to perform, as described in AARC-G027[https://aarc-community.org/guidelines/aarc-g027/]

Reply:
Done